ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -232,6 +278,7 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        container.addArrangedSubview(searchProgressSpinnerSwitchView)
```

Comment:
>container.addArrangedSubview(searchProgressSpinnerSwitchView) [](start = 8, length = 61)

don't show when we do not have accessory view

Reply:
Fixed. Added Screenshots for view without accessory

---
In reply to: [470798570](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470798570) [](ancestors = 470798570)